### PR TITLE
drawer_enhancement

### PR DESCRIPTION
**Pull request by trupti123-sahu**
I have given border to the container in drawer
changed the icons 
wrapped the listile with card

### CODE REVIEW

**Comment by trupti123-sahu** [comment]
WhatsApp Image 2021-01-31 at 20 50 24

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```
Comment (smaranjitghose) [inline .vscode/launch.json:3]: Delete these comments please

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (smaranjitghose) [inline .vscode/launch.json:4]: Delete these comments please

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 2 above)
Comment (trupti123-sahu) [reply .vscode/launch.json:4]: Sir I have not done anything in launch.json . These are auto-generated lines in vs code I think.

Comment (anushbhatia) [reply .vscode/launch.json:4]: remove these lines in the comment rest everything is fine

**Pull request closed without merging**